backend/vxlan/vxlan.go

```diff
@@ -164,28 +161,28 @@ func (o *Overlay) configure() error {
 	peersNetworks, linkedPeersContainers := getLinkedPeersInfo(allServices, selfService, networksMap, selfNetwork)
```

Comment:
Can you please test linking two environments to make sure HNA use case still works?